src/test/scala/com/wix/mysql/ParallelEmbeddedMysqlTest.scala

```diff
@@ -0,0 +1,29 @@
+package com.wix.mysql
+
+import com.wix.mysql.EmbeddedMysql._
+import com.wix.mysql.config.MysqldConfig.{SystemDefaults, aMysqldConfig}
+import com.wix.mysql.distribution.Version
+import com.wix.mysql.support.IntegrationTest
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+class ParallelEmbeddedMysqlTest extends IntegrationTest {
+
+  "EmbeddedMysql instance" should {
+
+    "run 2 instances in parallel" in {
+      withCleanRepo {
```

Comment:
I was thinking about it, but decided to delay this specialization and leave operation generic for now. I expect for more usages to happen soonish, then I will see.